<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>880</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <weight>50</weight>
    <bold>false</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">backgrouong_color:rgqconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(255, 255, 255, 255), stop:0.373979 rgba(255, 255, 255, 255), stop:0.373991 rgba(33, 30, 255, 255), stop:0.624018 rgba(33, 30, 255, 255), stop:0.624043 rgba(255, 0, 0, 255), stop:1 rgba(255, 0, 0, 255))rgb(255, 255, 255)b(255, 255, 255)
</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="Button_bro">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_close">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>850</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭窗口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_picAdr">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>图片地址...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_recognize">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>450</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>识别文字</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_img">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>371</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string>原始图像</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_text">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>识别结果（可复制！）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>810</y>
     <width>531</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background_color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>提示：本软件调用百度OCR(文字识别库和通用翻译库)，每天识别次数限量，如无法识别，请周知！（赵明2018.3.25）</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_translate">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>760</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>翻译</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_chooseLanguage">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>760</y>
     <width>87</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>English</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit_translateLanguage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>421</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_fromLang">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>760</y>
     <width>87</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Chinese</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Button_close</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
